pyramid/tier_fs_test.go

```diff
@@ -138,7 +148,6 @@ func TestStartup(t *testing.T) {
 	bytes, err := ioutil.ReadAll(f)
 	require.NoError(t, err)
 	require.Equal(t, content, string(bytes))
-
 }
 
 func testEviction(t *testing.T, namespaces ...string) {
```

Comment:
This is used from multiple places, so AFAIU it needs to have a `t.Run()` somewhere around it -- either inside or at the call site.  Otherwise it's just harder to tell what happened.

Reply:
Not sure I follow - this is a utility function. If it fails any test (which it could), the failing test name will appear in the log.
What information is missing in your opinion?

Reply:
Synced f2f.